io/src/hdl_grabber.cpp

```diff
@@ -542,7 +542,6 @@ pcl::HDLGrabber::stop ()
 
   if (hdl_read_packet_thread_ != NULL)
   {
-    hdl_read_packet_thread_->interrupt ();
```

Comment:
interrupt() does not exist in standard C++, but this is also not required since there is an exit condition for the hdl_read_packet_thread_